src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -54,9 +54,9 @@
     <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.6" />
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.0.8100001-0126c21e" />
     <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="3.0.13" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Logging" Version="3.0.13" />
-	  <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.3" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Host.Storage" Version="3.0.14" />
```

Comment:
Temporary changes for local testing. Once the Core SDK changes are in, I'll update this to pull the actual version.